web/pages/api/v0/market/[id].ts

```diff
@@ -25,6 +28,8 @@ export default async function handler(
     'userId'
   >[]
 
+  const answers = (contract as FreeResponseContract).answers
```

Comment:
I think it would be more idiomatic Typescript to sort of be more explicit rather than typecasting:
```typescript
const answers = contract.outcomeType === 'FREE_RESPONSE' ? contract.answers : undefined;
```
For example, that would result in `answers` having the correct type of `Answer[] | undefined`, instead of the incorrect type of `Answer[]`.